src/metabase/driver/crate/native.clj

```diff
@@ -0,0 +1,31 @@
+(ns metabase.driver.crate.native
+  (:require [clojure.java.jdbc :as jdbc]
+            [metabase.models.database :refer [Database]]
+            [metabase.db :refer [sel]]
+            [metabase.driver.generic-sql :as sql]
+            [clojure.tools.logging :as log]
+            [metabase.util :as u]
+            [metabase.driver.generic-sql.native :as n]))
+
+(defn process-and-run
```

Comment:
@agilliland no not explicitly for the connection like postgresql does. However, in the `date_trunc` scalar function we are supporting timezones (see https://crate.io/docs/reference/sql/scalar.html#date-trunc-interval-timezone-timestamp-returns-timestamp). If its not explicitly specified it is always based on UTC - like it is now.
I will discuss that internally if we provide it there or not.